### PR TITLE
doc: Update remove repo page

### PR DESCRIPTION
**Pull request by indradhanush**
## Test plan

1. `sg start` or `sg run docsite`
1. http://localhost:5080/admin/how-to/remove-repo

image

### CODE REVIEW

File: doc/admin/how-to/remove-repo.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+- site-admin level permissions on your Sourcegraph instance
+- access to your Sourcegraph deployment
 
 ## Steps to remove a repository from Sourcegraph
 
-1. Add the repository name to the [exclude list](https://docs.sourcegraph.com/admin/external_service/github#exclude) in your [code host configuration](https://docs.sourcegraph.com/admin/external_service).
-1. Wait for the repository to disappear from the Repository Status Page located in your Site Admin panel.
+Open the repository's `Settings` page on Sourcegraph and from the `Options` tab click `Exclude repository` to exclude it from the specific code host.
+
+![Exclude repository from code host](https://storage.googleapis.com/sourcegraph-assets/docs/images/admin/how-to/exclude-repo.png)
```
Comment (mrnugget) [inline doc/admin/how-to/remove-repo.md:14]: Not blocking, but just for future reference: I try to run assets in docs through `ImageOptim.app` before uploading them, just to save some kilobytes. https://imageoptim.com/mac

**Review by mrnugget — APPROVED** [review]
Very good changes!

File: doc/admin/how-to/remove-repo.md (hunk 1 above)
Comment (indradhanush) [reply doc/admin/how-to/remove-repo.md:14]: Nice. TIL. I will run the assets through the app and upload them again. Nothing should change in this PR and it should continue to work. 👍🏽

Comment (indradhanush) [reply doc/admin/how-to/remove-repo.md:14]: Good savings. 
image

Comment (indradhanush) [reply doc/admin/how-to/remove-repo.md:14]: Uploaded trimmed assets to Google storage. ✅

**Pull request merged**